spec/models/component_spec.rb

```diff
@@ -127,5 +129,45 @@ def expected_config(event_id)
         ]
       }
     end
+
+    it 'does not include expired signing certs' do
+      travel_to Time.now + 2.months + 2.days
+
+      msa_component.services << msa_service
+      sp_component.services << sp_service
```

Comment:
Since we're doing this three times now I think we should add this to the before each and take them out of all the it blocks.

```
before(:each) do
  SpComponent.destroy_all
  MsaComponent.destroy_all
  msa_component.services << msa_service
  sp_component.services << sp_service
end
```